scripts/polkadot-launch/composable.json

```diff
@@ -82,19 +82,19 @@
           "wsPort": 9188,
           "port": 33200,
           "basePath": "/tmp/polkadot-launch/parachains/dali-2-alice",
-          "flags": ["--alice", "--rpc-cors=all", "--execution=wasm", "--wasmtime-instantiation-strategy=recreate-instance-copy-on-write", "--log=pallet_ibc=trace"]
+          "flags": ["--alice", "--rpc-cors=all", "--execution=wasm", "--wasmtime-instantiation-strategy=recreate-instance-copy-on-write", "--log=pallet_ibc=trace", "--enable-offchain-indexing=true"]
         },
         {
           "wsPort": 9189,
           "port": 33201,
           "basePath": "/tmp/polkadot-launch/parachains/dali-2-bob",
-          "flags": ["--bob", "--rpc-cors=all", "--execution=wasm", "--wasmtime-instantiation-strategy=recreate-instance-copy-on-write"]
+          "flags": ["--bob", "--rpc-cors=all", "--execution=wasm", "--wasmtime-instantiation-strategy=recreate-instance-copy-on-write", "--enable-offchain-indexing=true"]
         },
         {
           "wsPort": 9190,
           "port": 33202,
           "basePath": "/tmp/polkadot-launch/parachains/dali-2-charlie",
-          "flags": ["--charlie", "--rpc-cors=all", "--execution=wasm", "--wasmtime-instantiation-strategy=recreate-instance-copy-on-write"]
+          "flags": ["--charlie", "--rpc-cors=all", "--execution=wasm", "--wasmtime-instantiation-strategy=recreate-instance-copy-on-write", "--enable-offchain-indexing=true"]
```

Comment:
for those who don't know (like me) what this is: https://docs.substrate.io/main-docs/fundamentals/offchain-operations/